include/uv-win.h

```diff
@@ -45,8 +45,8 @@ typedef struct pollfd {
 
 #ifndef _malloca
 # if defined(_DEBUG)
-#  define _malloca(size) malloc(size)
-#  define _freea(ptr) free(ptr)
+#  define _malloca(size) uv__malloc(size)
+#  define _freea(ptr) uv__free(ptr)
```

Comment:
I agree, I think it can be scrapped.  It'd be nice to have confirmation from @piscisaureus though.